ui/src/app/domains/views/DomainsList/DomainListHeaderForm/DomainListHeaderForm.test.tsx

```diff
@@ -3,41 +3,39 @@ import { act } from "react-dom/test-utils";
 import { Provider } from "react-redux";
 import configureStore from "redux-mock-store";
 
-import DomainListHeader from "../DomainListHeader/DomainListHeader";
+import DomainListHeaderForm from "./DomainListHeaderForm";
```

Comment:
@ClementChaumel generally we try to only run tests on the component in that directory. There are some older form components in the codebase that do the same as here however - just ignore those. :)